akka-actor-typed/src/main/protobuf/EntityMessages.proto

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2014-2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+syntax = "proto2";
+
+package akka.actor.typed;
+
+option java_package = "akka.actor.typed.internal.protobuf";
+option optimize_for = SPEED;
+
+
+// keep this in sync with Payload from ContainerFormats.proto in akka-remote
+message Payload {
+  required bytes enclosedMessage = 1;
+  required int32 serializerId = 2;
+  optional bytes messageManifest = 4;
+}
+
+message EntityEnvelope {
+  required string entityId = 1;
+  required Payload message = 2;
+}
```

Comment:
I added this proto because this needs to go through the wire, but in the meantime I think we can remove it by unwrapping the EntityEnvelope and building a ShardingEnvelope to send. I didn't work this out to the end, so for the moment still depending on this.